### DISCUSSION TITLE
Sending only errors but not the errorless transactions

### DISCUSSION THREAD

**Discussion by minicode01** [getsentry/support#15] [Support] [answered]
Dear Sentry community,

I am using sentry on a Python/Flask based website. This is a great tool with amazing features.  My goal is to be informed whenever a problem happens with the code -along with the current context- causing an error in the application, which the internet user would have never reported.

Despite having no errors for a couple of weeks, I am quickly running out of "transactions". The quota is being eaten even without errors reported.

I understand the "**traces_sample_rate**" can be lowered to reduce the percentage of transactions reported. What is unclear is: let's say the rate is reduced to 30%. And an error occurs once in a week period (because of very specific set of conditions). Is it possible that the error will occur during the 70% unreported, and will therefore never make it to the Sentry.IO portal? 

How can we configure the SDK in such a way that only errors are reported, and not the errorless transactions, so as not to eat all the quota so quickly?

Thanks a lot.

**Comment by Kyoss79** [comment]
Even when you set the traceSampleRate to zero, sentry is sending tons of requests to its API.
Don't know why.

**Comment by silent1mezzo** [comment, marked as answer]
Hey @minicode01, `traces_sample_rate` will only affect the number of transactions that get sent to Sentry, not the errors. For example, setting `traces_sample_rate = 0.3` you'll still send 100% of your errors.

There is a separate option for sampling errors ([`sample_rate`](https://docs.sentry.io/platforms/python/guides/beam/configuration/options/#sample-rate)) but by default all errors will be sent regardless of the `traces_sample_rate`.

**Reply by minicode01** [reply]
Thank you for taking the time to write this answer :)
I updated my configuration to 0.2 to see if this helps, and report back here.
